history/cultures/draenei.txt

```diff
@@ -54,5 +54,8 @@
 605.1.1 = {
 	discover_innovation  = innovation_hoardings 
 	#
-	discover_innovation = innovation_heraldry 
+	discover_innovation = innovation_heraldry
+}
+610.1.1 = {
+	discover_innovation = innovation_elephantry
```

Comment:
It's discovered twice.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/111865170-dbb6b180-897e-11eb-8c02-c90781fc859f.png)

</details>